store/index.js

```diff
@@ -67,6 +67,14 @@ const initialState: StateType = {
 			focused: false,
 		},
 		{ ...codeBlockInstance, focused: false },
+		{
+			uid: '5',
+			name: 'core/more',
+			isValid: true,
+			attributes: { customText: undefined },
+			innerBlocks: [],
+			focused: false,
+		},
```

Comment:
Can we reuse the `createBlock()` method to create the block (see `codeBlockInstance` earlier for example) to stay close to the original code generation? `createBlock` should ideally support the `core/more` block at this point.

Reply:
Good point, fixed in 359eae6